config/samples/operators_v1alpha1_operator.yaml

```diff
@@ -9,4 +9,4 @@ metadata:
     app.kubernetes.io/created-by: operator-controller
   name: operator-sample
 spec:
-  # TODO(user): Add fields here
```

Comment:
Keep the TODO comment for future field adders?

Reply:
Sure!

Reply:
Updated